### PR TITLE
Initial implementation

### PR DESCRIPTION
**Pull request by bajtos**
Implement a new script "lb" that defers to generator-loopback under the hood.

Available commands (loopback generators):

```
  lb acl
  lb app
  lb boot-script
  lb datasource
  lb export-api-def
  lb middleware
  lb model
  lb property
  lb relation
  lb remote-method
  lb swagger
```

Additional functionality:

```
$ lb --version
0.1.0 (generator-loopback@2.0.0)
```

Out of scope: README, IBM-related messaging.

Connect to strongloop-internal/scrum-loopback#1028
Requires https://github.com/strongloop/generator-loopback/issues/255

@superkhau @ritch please review
cc @chandadharap

### CODE REVIEW

**Comment by bajtos** [comment]
> ` [cis-jenkins] x64 && linux && nvm,6 — Failed! (e8c044c)`

I am afraid the inquirer-output parsing is still not robust enough (even after the three iterations I made). I'll look into this problem next week, I think the first step is to figure out how to provide a more verbose error message when the prompt-parsing algorithm fails.

Please don't withhold the review!

File: test/helpers/responder.js [hunk 1]
```diff
@@ -0,0 +1,127 @@
+// copyright ibm corp. 2016. all rights reserved.
+// node module: loopback-cli
+// this file is licensed under the mit license.
+// license text available at https://opensource.org/licenses/mit
```
Comment (superkhau) [inline test/helpers/responder.js]: Uppercase the first chars like the other comments.

File: test/helpers/responder.js [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+}
+
+function detectPromptAndRespond(chunk) {
+  chunk = this.handleInquirerScreenManipulation(chunk);
+  if (!chunk) return;
+  chunk = chunk.replace(/\s+/g, ' ').trim();
+  if (!chunk) return;
+
+  for (var ix in PROMPTS) {
+    // const [regexp, action] = PROMPTS[ix];
```
Comment (superkhau) [inline test/helpers/responder.js]: remove unused comment?

File: test/smoke.test.js [hunk 3]
```diff
@@ -0,0 +1,285 @@
...
+          .to.have.property('children')
+          .eql({
+            type: 'hasMany',
+            model: 'node',
+            foreignKey: 'parentId',
+          });
+      });
+  });
+
+  // The following generatores require the project to have all dependencies
```
Comment (superkhau) [inline test/smoke.test.js]: generatores -> generators

**Comment by bajtos** [comment]
@superkhau I addressed your comments and fixed the test suite to be more robust. All tests are green now, yay! PTAL at the new commits.

**Comment by ritch** [comment]
Also LGTM - and this code looks pretty familiar (its mostly been reviewed already).

**Comment by bajtos** [comment]
Landed 🎉

**Pull request merged**